### ISSUE TITLE
Needs an Offline Installer

### ISSUE THREAD

**Issue by yndrelbosch** [TanninOne/modorganizer#833]
I can't install this pos because I have no internet on my home pc, and as much as I love MO, it's kind of hard to use it when it won't install without connecting to the internet to download stuff. and Yes, I saw a post earlier saying soemthing about downloading the packages separately and all that, but when I try that, and start up MO it says can't start due to some issue with side by side configuration.

PS. If I sound annoyed or angry, well, I'm sorry, but I've been having a craptastic day, and I am trying very hard not to pick up my stupid computer and throw it out a four story window ... ~~

**Comment by RPINerd** [comment]
I don't know for sure.. but I think there's an option to install it as a "portable" install. If you do that on a thumb drive somewhere where you have internet access and then transfer it to your desktop I believe the main thing you would need to do is just modify some of the folder directories.

I could be completely wrong about that but it would be easier than trying to relocate your desktop XD

**Comment by yndrelbosch** [comment]
thanks, I'll try that, and sorry for my tone last night, I was really really upset with my computer XD

**Comment by Occams-Cat** [comment]
Completely uninstallable, as there's no "Portable" option.
This gets part way through installing, then tells me the installation failed because the installation program couldn't "Connect" to something.

Can't give any meaningful feedback on a program I can't install and use.

**Comment by RPINerd** [comment]
So if I was a betting man, I'd say you have a firewall issue. I've never had an issue installing MO except for the one time I tried to install it on my companies network. At work basically everything "inappropriate" is blocked and when I try to install I get that message.

Is your PC on a network managed by someone who isn't you? Something like a landlord or maybe a community access point in a block of flats?

**Comment by Occams-Cat** [comment]
No, I manage my own PC, and even with my firewall and AV completely shut off it won't install.
